analysis_options.yaml

```diff
@@ -18,3 +18,4 @@ analyzer:
   exclude:
     - lib/**/*.g.dart
     - lib/**/*.gr.dart
+    - lib/assets/**
```

Comment:
better place empty line to the end of file

Reply:
\+